@app/worker/package.json

```diff
@@ -3,12 +3,12 @@
   "private": true,
   "version": "0.0.0",
   "scripts": {
-    "gw": "cd dist && npx --no-install -n \"-r @app/config/env\" graphile-worker",
+    "gw": "cd dist && cross-env NODE_OPTIONS=\"$NODE_OPTIONS -r @app/config/env\" graphile-worker",
```

Comment:
To use `cross-env` here, we need to move it from `devDependencies` to `dependencies`.